protocols/nsclient/src/test/java/org/opennms/protocols/nsclient/config/NSClientPeerFactoryTest.java

```diff
@@ -70,6 +78,31 @@ public final void testOneSpecific() throws IOException {
         assertEquals(1, factory.getConfig().getDefinition().size());
     }
 
+    @Test
+    public final void testMetadata() throws IOException {
+        final TemporaryFolder tempFolder = new TemporaryFolder();
+        tempFolder.create();
+        final File keystoreFile = new File(tempFolder.getRoot(), "scv.jce");
+        final SecureCredentialsVault secureCredentialsVault = new JCEKSSecureCredentialsVault(keystoreFile.getAbsolutePath(), "notRealPassword");
+        secureCredentialsVault.setCredentials("nsclient", new Credentials("foo", "bar"));
+
+        final String amiConfigXml = "<?xml version=\"1.0\"?>\n" +
```

Comment:
Perhaps rename to `nsclientConfigXml` or just `configXml`.

Reply:
done.